README.md

```diff
@@ -37,9 +37,15 @@ Cases where the RFC process need *not* be used:
     * Risks
  - The proposal is discussed by the community and it is assumed that the
    proposal will change in accordance with that discussion.
- - In the case of no substantive disagreement the RFC is considered accepted
-   after 1 week. If any participant requests it, the comment period is extended
-   to 2 weeks.
+ - At least one review approval is required to merge a PR. For changes that
+   may impact downstream WPT consumers more approvals should be sought, or at
+   least a clear acknowledgement from those consumers that this change can be
+   accommodated in their workflows.
+ - If reviewers or downstream consumers are not responsive after at least two
```

Comment:
Does one need to wait two weeks, or is it OK to put it on the agenda for the next WPT infra meeting when the RFC is filed?

Reply:
No need to wait, happy to reword this to make it clearer.

Reply:
Sounds good! Something like feel free to add it to the agenda any time you see fit, but in particular if nobody is responding to the RFC.

Reply:
Thanks, I reworded this section to capture your suggestion.